src/palakSJun20/User.java

```diff
@@ -0,0 +1,81 @@
+/*Exam-7:
+Q1) class User{
+void displayInfo(){
+debit
+credit
+}
+
+void displayAllInfo(){
+totaldebit
+totalCredit
+totalUsers
+}
+}
+
+output : Users -> 3
+User1 -> total debit
+User1 -> total credit
+
+Total --> total debit & total credit
+*/
+package palakSJun20;
+
+public class User {
+	static int objectCount;
+	int userCreditCount, userDebitCount;
+	static int totalCreditCount, totalDebitCount;
+	String name;
+	public User(String name) {
+		objectCount++;
+		this.name = name;
+	}
+	
+	public String toString() {
+		return this.name;		
+	}
+	
+	void debit() {
+		userDebitCount++;
+		totalDebitCount++;
+	}
+	
+	void credit() {
+		userCreditCount++;
+		totalCreditCount++;
+	}
+	
+	void displayInfo() {
+		System.out.println(toString() + " --> Debit count is " + userDebitCount);
```

Comment:
instead of toString(), cant we directly say name.
toString method should be override when you want to print User detail when you print referece.